devtools-ci-index.yaml

```diff
@@ -2415,13 +2415,13 @@
             sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
             ssh_cmd="ssh $sshopts $CICO_hostname"
             $ssh_cmd yum -y install rsync
-            rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
+            rsync --delete -e "ssh $sshopts" -Ha $(pwd)/booster_bdd/ $CICO_hostname:payload/booster_bdd/
```

Comment:
I see other templates using `-delete` is it typo and does not work?

Reply:
IMHO, `rsync` does not have `-delete` parameter, it's just a synonym for `-d -e -l -e -t -e`, right? But it has `--delete`. So my guess is that other templates have typo in the respective command and it does not work as expected.

It (`-delete`) certainly does not work at my laptop (Fedora 28).

Reply:
so maybe it worth to update all places where we have wrong flag? can you do that?

Reply:
Sure, here it is: https://github.com/openshiftio/openshiftio-cico-jobs/pull/768